tools/editor/io_plugins/editor_scene_import_plugin.cpp

```diff
@@ -656,6 +658,9 @@ void EditorSceneImportDialog::_choose_file(const String& p_path) {
 
 
 	import_path->set_text(p_path);
+	if (root_node_name->get_text().size()==0){
+		root_node_name->set_text(import_path->get_text().get_file().basename());
```

Comment:
I was actually thinking about `LineEdit::set_placeholder`: https://github.com/godotengine/godot/blob/master/scene/gui/line_edit.cpp#L934-L938

It shows a placeholder text (with lower alpha value) that you can overwrite by typing, that fitted well with your previous implementation. But pre-filling "for real" the text is fine too IMO, I don't have a strong opinion about which is best.